tauri-app/src-tauri/src/seventh_heaven.rs

```diff
@@ -66,7 +90,15 @@ pub(crate) async fn install_run(app_handle: AppHandle) -> Result<(), ()> {
     }).unwrap();
 
     with_status(&app_handle,"Setting up FF7...".to_string(), || -> io::Result<()> {
-        todo!("Fetch FF7 and install it");
+        let game_path = steam.get_game_path(39140);
+        if !SteamManager::can_read_path(&game_path) {
+            panic!("We can't read the game path at {:?}", game_path)
+        }
+        match copy_directory(game_path.as_path(), &wine_manager.get_c_path("FF7")) {
+            Ok(_) => Ok(println!("We copied to {:?} successfully!",
```

Comment:
Should use the logger methods, such as `info!()`

Reply:
Okay, I fixed this